semesterly/settings.py

```diff
@@ -359,3 +359,19 @@ def get_secret(key):
     DEBUG_TOOLBAR_CONFIG = {"SHOW_TOOLBAR_CALLBACK": lambda request: True}
     INSTALLED_APPS += ("debug_toolbar",)
     MIDDLEWARE += ("debug_toolbar.middleware.DebugToolbarMiddleware",)
+    DEBUG_TOOLBAR_PANELS = [
```

Comment:
What do these do?

Reply:
It lets you configure what panels to show in debug toolbar and their orders